woocommerce/woocommerce-ios
# Add UI Test - Verify Products list screen

### Description
This PR adds a new UI test that checks and validates that the product list screen is listed and the first item on screen matches the data provided. In this PR, the only check is on the product name. Have not added for the other text/labels on the screen because there are currently no accessibilityIdentifiers on the cell elements and I'm not sure where/how to add them. We can add checks for the rest of elements after adding the accessibilityIdentifiers but that should be on different PR. 

### Testing instructions
Ensure that `testProductsScreenLoad()` passes and fails when it should (can make it fail by selecting the second item on the list instead of the first)

---
